locale/en/foundation/case-studies.md

```diff
@@ -16,7 +16,7 @@ preferences (current and future) for front end, back end and IoT developers.
 
 * [Read the full announcement.](/en/blog/announcements/nodejs-foundation-survey/)
 * [See the infographic.](/static/documents/2016-survey-infographic.png)
-* [Read the full report.](/static/documents/2016-survey-report.pdf)
+* <a href="/static/documents/2016-survey-report.pdf" onClick="ga('send', 'event', { eventCategory: 'casestudy', eventAction: 'click', eventLabel: 'Node.js 2016 Report', eventValue: 0});">Read the full report.</a>
```

Comment:
No, you just need to add a link like `<a href="filename.pdf" data-casestudy="Case Study Title">Read the full report.</a>` that's all. Everything else will happen automatically.

/cc @nodejs/website Any thoughts about this?